src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -1170,6 +1170,11 @@ void MacroAssembler::addpd(XMMRegister dst, AddressLiteral src) {
   }
 }
 
+// See 8273459.  Function for ensuring 64-byte alignment, intended for stubs only.
```

Comment:
I suggest to extend comment to explain why it will not for nmethods - because they are copied when published.